modules/javascript/web-worker/assets/js/partytown/debug/partytown.js

```diff
@@ -0,0 +1,72 @@
+/* Partytown 0.5.1 - MIT builder.io */
+!function(win, doc, nav, top, useAtomics, config, libPath, timeout, scripts, sandbox, mainForwardFn, isReady) {
+    function ready() {
+        if (!isReady) {
+            isReady = 1;
+            libPath = (config.lib || "/~partytown/") + (false !== config.debug ? "debug/" : "");
+            if ("/" == libPath[0]) {
+                scripts = doc.querySelectorAll('script[type="text/partytown"]');
+                if (top != win) {
+                    top.dispatchEvent(new CustomEvent("pt1", {
+                        detail: win
+                    }));
+                } else {
+                    timeout = setTimeout(fallback, 1e4);
+                    doc.addEventListener("pt0", clearFallback);
+                    useAtomics ? loadSandbox(1) : nav.serviceWorker ? nav.serviceWorker.register(libPath + (config.swPath || "partytown-sw.js"), {
+                        scope: libPath
```

Comment:
This will be compatible with the PWA plugin as long as the `scope` is defined as the actual library path and not `/`.